pytorch_lightning/core/lightning.py

```diff
@@ -162,6 +164,26 @@ def automatic_optimization(self) -> bool:
         """
         return True
 
+    @property
+    def decision_on_invalid_result(self) -> Union[str, int]:
```

Comment:
new users do not know about the internal result object. call it `invalid_loss_strategy`?

Reply:
also return should be just `str`

Why did you not end up using an Enum?

`class InvalidLossStrategy(str, Enum)`

Reply:
It is a LightningEnum. It should work with both str and Enum. I used the enum, so people can go to definition if they look into the code.

Reply:
Renamed `invalid_loss_strategy`.